_health-care/_js/components/insurance-information/MedicareMedicaidSection.jsx

```diff
@@ -27,11 +29,10 @@ class MedicareMedicaidSection extends React.Component {
         </div>
         <div className="row">
           <div className="small-12 columns">
-            <input
-                id="veteran_is_enrolled_medicare_part_a"
-                name="veteran_is_enrolled_medicare_part_a"
-                type="checkbox"/>
-            <label htmlFor="veteran_is_enrolled_medicare_part_a">Are you enrolled in medicare hospital insurance part a?</label>
+            <ErrorableCheckbox
+                label="Are you enrolled in medicare hospital insurance part a?"
```

Comment:
Same here, I think it should be `Medicare Part A (hospital insurance)` (from Google).